activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/api/task/TaskQueryTest.java

```diff
@@ -2661,6 +2661,33 @@ public void testLocalizeTasks() throws Exception {
     assertThat(task.getDescription()).isEqualTo("My 'en' localized description");
   }
 
+    public void testQueryCandidateOrAssignedUser() {
+        // given a new task with an assigned user
```

Comment:
adding another task here would a good idea. The test will fail if more than one task is returned in the query